extensions/clusters/clusters.go

```diff
@@ -868,32 +856,31 @@ func HardenK3SClusterConfig(clusterName, namespace string, clustersConfig *Clust
 	return v1Cluster
 }
 
-// HardenRKE2ClusterConfig is a constructor for a apisV1.Cluster object, to be used by the rancher.Client.Provisioning client.
+// HardenRKE1ClusterConfig is a function that modifies the cluster configuration to be hardened according to the CIS benchmark.
+func HardenRKE1ClusterConfig(client *rancher.Client, clusterName string, clustersConfig *ClusterConfig) *management.Cluster {
+	cluster := NewRKE1ClusterConfig(clusterName, client, clustersConfig)
+
+	cluster.DefaultPodSecurityAdmissionConfigurationTemplateName = rancherRestricted
+	cluster.RancherKubernetesEngineConfig.Services.Etcd.GID = 52034
```

Comment:
seeing these as raw values is strange to me. Can you make it a const and maybe even add a comment as to why it is what it is, or a ref in the rancher/rancher code to where this is set? might be overkill but if we ever have to change it, this info would be nice to have.

Reply:
I can put it into a const. I personally think it's too much to comment it. The hardening guides we put out to customers go into extreme detail as to all of this, so I would expect someone to read that.